app/src/main/js/components/rants.js

```diff
@@ -1,11 +1,41 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
```

Comment:
So basically, a `component` can be written as a pure function in every case. `Components` do not have a `state` nor do they have any knowledge of `redux`, be it the store or the dispatching mechanism. 
If you have UI elements that do need a `state` or access to the `store`, it is generally considered a `container` and should be moved into the folder.